packages/core/src/constants.ts

```diff
@@ -37,3 +37,14 @@ export const CURRENT_CHUGSPLASH_MANAGER_VERSION = {
   minor: 0,
   patch: 0,
 }
+
+export const LAYERZERO_ENDPOINT_ADDRESSES = {
+  5: {
+    address: '0xbfD2135BFfbb0B5378b56643c2Df8a87552Bfa23',
+    lzChainId: 10121,
+  },
+  420: {
+    address: '0xae92d5aD7583AD66E49A0c67BAd18F6ba52dDDc1',
+    lzChainId: 10132,
+  },
+}
```

Comment:
LayerZero does have a package with addresses, but I don't think we should use it for a couple of reasons:
1. It's an alpha package that they warn could be unstable
2. It's distributed under a business license (disallows commercial use) which is super weird tbh